Client/game_sa/gamesa_function_caller.h

```diff
@@ -0,0 +1,52 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        game_sa/gamesa_function_caller.h
+ *  PURPOSE:     Game functions caller
+ *
+ *  Multi Theft Auto is available from https://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#pragma once
+#include "StdInc.h"
```

Comment:
I think you should not use precompiled header here

Reply:
Why?